ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/gui/BoardPanel.java

```diff
@@ -50,17 +52,36 @@ public BoardPanel(BoardSize boardSize, int boardIndex, GlassPane globalGlassPane
         };
 
         setOpaque(false);
-        setLayout(new GridLayout(boardSize.width + 1, boardSize.height));
-        setPreferredSize(new Dimension((boardSize.width + 1) * 48, (boardSize.height + 1) * 48));
-        createGrid(boardSize);
+        setLayout(new GridLayout(mBoardSize.width + 1, mBoardSize.height));
+        setPreferredSize(new Dimension((mBoardSize.width + 1) * 48, (mBoardSize.height + 1) * 48));
+        createGrid();
     }
 
-    private void createGrid(BoardSize boardSize) {
-        for (int y = boardSize.height; y > 0; y--) {
+    public void rescaleBoard(int panelWidth, int panelHeight){
+        Double boardScale = (panelHeight * .85);
```

Comment:
Only use `Double` when you're concerned about the value being null. Lowercase `double` is safe here